src/app/dashboard/page.tsx

```diff
@@ -343,11 +403,12 @@ const Home: NextPage = () => {
             </Box>
           </Flex>
         </GridItem>
+
         <GridItem colSpan={{ base: 12, lg: 12 }}>
           <Box mb={6}>
             <GraphComponent
               graphData={treasuryGraphData}
-              graphCurrentValue={getCurrentHolderValue(treasuryGraphData)}
```

Comment:
Why remove this function?

Reply:
I removed it since the resolve was having the same data displayed on both instances(dashboard and treasury page), plus the current value it's running is way lesser than the total value that needs to be shown.